src/test/kotlin/org/ionproject/integration/file/PDFFileDownloaderTest.kt

```diff
@@ -0,0 +1,90 @@
+
+
+package org.ionproject.integration.file
+import java.net.ConnectException
+import java.nio.file.FileSystems
+import java.nio.file.Path
+import java.nio.file.PathMatcher
+import java.nio.file.Paths
+import org.ionproject.integration.file.exceptions.InvalidFormatException
+import org.ionproject.integration.file.exceptions.ServerErrorException
+import org.ionproject.integration.file.implementations.PDFFileDownloader
+import org.ionproject.integration.file.interfaces.FileDownloader
+import org.ionproject.integration.utils.Try
+import org.ionproject.integration.utils.orThrow
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+internal class PDFFileDownloaderTest {
+    companion object {
+        private val pdfDownloader: FileDownloader = PDFFileDownloader()
```

Comment:
Why is this in the companion object?

Reply:
Without it being in the companion object, method downloadPdf cannot use it